catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/ListMetacardTypeImpl.java

```diff
@@ -38,23 +38,7 @@
   static {
     LIST_DESCRIPTORS = new HashSet<>();
 
-    LIST_DESCRIPTORS.add(
-        new AttributeDescriptorImpl(
-            Metacard.ID,
-            true /* indexed */,
-            true /* stored */,
-            false /* tokenized */,
-            false /* multivalued */,
-            BasicTypes.STRING_TYPE));
-
-    LIST_DESCRIPTORS.add(
-        new AttributeDescriptorImpl(
-            Metacard.TITLE,
-            true /* indexed */,
-            true /* stored */,
-            false /* tokenized */,
-            false /* multivalued */,
-            BasicTypes.STRING_TYPE));
+    LIST_DESCRIPTORS.addAll(MetacardImpl.BASIC_METACARD.getAttributeDescriptors());
```

Comment:
❓ Any reason we don't take specifically what we need like in `NoteAttributes`?

Reply:
I guess I got lazy here :) Will fix.